src/models/UniqueNamedContainer.java

```diff
@@ -53,7 +51,13 @@ public UniqueNamedContainer() {
     public void add(T item) {
         T newItem = (T) item.getCopy();
 
-        if (!item.isGlobal()) {
+        if (item.isGlobal()) {
+            // If the item we are adding is global then we only want to add it if it's unique name is not present.
```

Comment:
```suggestion
            // We only want to add global items it if its unique name is not present.
```
There were too many ifs for me to parse the sentence on the first go. And the classical it's vs its 😆.